data/xcspecs.bzl

```diff
@@ -27,26 +27,6 @@ SETTINGS = {
         "Name": "Apple Clang",
         "OptionConditionFlavors": ["arch", "sdk"],
         "Options": {
-            "CLANG_TARGET_TRIPLE_ARCHS": {
```

Comment:
is this removal temporary?
also wondering how other downstream projects will be affected, consider adding @tinder-maxwellelliott as a reviewer for example

Reply:
We can add some of these flags back if we end up implementing a comprehensive xcconifg evaluator, which would be more involved and perhaps ideal.

It's unlikely a user would set these, mainly if they did today it'd be broken - I suspect it's a safe change.

Reply:
Instead of manually modifying this file, should we update the xcspec_extractor.py and run `bazel run data_generators:extract_xcspecs`? We can also add some comments there.

Reply:
Agree with Cong's comment. Directly editing codegen seems undesirable.

How much work would it be to handle these fields, rather than sidestepping these values in the xcconfig? That seems like it would be directionally better.

Will this leave the values in the xcconfig applied in the generated project (even if Bazel build settings are fine)? Will that lead to unexpected IDE behavior?

Reply:
+1 

This will also make this and the other generated file out of sync: https://github.com/bazel-ios/rules_ios/blob/b439882984e3e756f8a54a589b6aa3e7071bb502/data/xcspec_evals.bzl#L3669

Reply:
Yep, it's a good point these are generated - we can probably pop them off inside of the method that processes these!